ToDoList/ToDoList/Modules/TaskDisplay/Core/Model/TaskItem.swift

```diff
@@ -7,21 +7,32 @@
 import UIKit
 
 enum TaskStatus: String, CaseIterable {
-    case active = "Active"
-    case completed = "Completed"
+    case active
+    case completed
+    
+    func initializeLocalization() -> String {
+        switch self {
+        case .active:
+            return TaskDisplayLocalization.tableViewActiveSectionTitle
+        case .completed:
+            return TaskDisplayLocalization.tableViewCompletedSectionTitle
+        }
+    }
 }
 
 struct TaskItem: Hashable {
     var status: TaskStatus
     var title: String
     var description: String?
+    var timeSinceLastChange: Date
```

Comment:
You can create the date when the object is initiated, instead of passing it in storage on initialisation
```suggestion
    var timeSinceLastChange = Date()
```